{"uuid": "d634a16f-b694-4f91-92f1-3ab39846f18f", "children": ["99e7989a-a15e-48d1-923e-d5246cf6fced"], "befores": [{"name": "login", "status": "passed", "start": 1598332838277, "stop": 1598332852418}], "afters": [{"name": "login::0", "status": "broken", "statusDetails": {"message": "EOFError: telnet connection closed\n", "trace": "  File \"d:\\program files\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\program files\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 796, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\CDATA\\自动化测试\\Cdata_part11\\tests\\FD1216S\\conftest.py\", line 31, in login\n    ont_del(tn, Epon_PonID, Epon_OnuID, Epon_ONU_MAC)\n  File \"E:\\CDATA\\自动化测试\\Cdata_part11\\src\\FD1216S\\ont_auth.py\", line 933, in ont_del\n    command_result = tn.read_until(b\"OLT(config-interface-epon-0/0)#\", timeout=2).decode('ascii')\n  File \"d:\\program files\\python37\\lib\\telnetlib.py\", line 327, in read_until\n    return self.read_very_lazy()\n  File \"d:\\program files\\python37\\lib\\telnetlib.py\", line 403, in read_very_lazy\n    raise EOFError('telnet connection closed')\n"}, "start": 1598336540083, "stop": 1598336540092}], "start": 1598332838277, "stop": 1598336540092}